packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -797,6 +797,11 @@ export type CallWithChatEvent = 'callError' | 'chatError' | 'callEnded' | 'isMut
 // @public
 export type ChatAdapter = ChatAdapterThreadManagement & AdapterState<ChatAdapterState> & Disposable & ChatAdapterSubscribers & FileUploadAdapter;
 
+// @beta
+export type ChatAdapterOptions = {
+    credential?: CommunicationTokenCredential;
```

Comment:
can we make the name less generic? We already use a credential to create the adapter/statefulchatclient (the ACS token credential), how does this credential differ?

Reply:
Perhaps something like this:
```
export type ChatAdapterOptions = {
    teamsInterop?: {
      credential: CommunicationTokenCredential;
    }
}
```

Reply:
I don't know if I agree with this change, adding extra layer. Now it feels like we are adding an options bag inside an options bag. I worry it might just adding unnecessary complexity

Reply:
Yeah, if we only have the one property lets keep flat 👍
Scope the name to something less generic to distinguish from the chat token credential and allow for other credentials if they're required; something like `teamsInteropFileDownloadCredential` ? (Can this be used for both file attachments and things like inline images?)

Reply:
Right now only inline images (specifically images coming in from teams) needs this credential. Future-wise is a little unknown. I would imagine going forward if we need to fetch anything from the service outside of the scope of ChatSDK (which is very niche) we would need it.

Reply:
Lets leave this as is then, if it needs changed can be easily done at api review time 👍